### PR TITLE
change my url

### PR DESCRIPTION
**Pull request by vpbharath**

### CODE REVIEW

**Comment by ERLtechtvm** [comment]
https://github.com/ERLtechtvm/RobotControl.git this is my changed url  can you please check

**Comment by ERLtechtvm** [comment]
https://github.com/ERLtechtvm/RobotControl.git this is my changed url can you please check @ArduinoBot

File: repositories.txt [hunk 1]
```diff
@@ -7218,5 +7218,4 @@ https://github.com/protheeuz/AsyncWebServerESP
 https://github.com/JoelJojoP/SimpleIMU
 https://github.com/vickash/RotaryEncoderPCNT
 https://github.com/kanitawa/RotEnc
-https://github.com/vpBharath/RobotControl.git
```
Comment (per1234) [inline repositories.txt:7221]: You must not remove the URLs of other people's libraries. That is incredibly irresponsible and disrespectful behavior.

Please add the URL back.

**Review by per1234 — COMMENTED** [review]
In addition to changing the URL, you also changed the library name.

The library name is defined by the `name` field in your library's `library.properties` metadata file:

https://github.com/ERLtechtvm/RobotControl/blob/main/library.properties#L1

```text
name=RobotControl
```

We do allow you to change the library name on request (though we discourage the practice as it makes work for us), but it is mandatory that every library have a unique name.

The new name you chose "RobotControl" has already been taken so you are not allowed to rename your library to this name.

Please change the value of the `name` field in your `library.properties` file to a unique name. You can search the [Arduino IDE Library Manager](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library#installing-a-library) to check whether a name is already taken.

You will need to do the following things:

1. Change the value of the `name` field in your `library.properties` file to a unique name.
   **ⓘ** You can search the [Arduino IDE Library Manager](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library#installing-a-library) to check whether a name is already taken.
1. Bump the value of the `version` field in the library's `library.properties` metadata file.
1. Create a new [release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository) or [tag](https://git-scm.com/docs/git-tag) in the library repository.
1. Add a comment here in the pull request thread that mentions **@ArduinoBot**.

After that, the automated checks will run again. Once those are passing, your library will automatically be added to Library Manager.

If you have any questions or concerns, please let me know.

Regards, Per

File: repositories.txt (hunk 1 above)
Comment (ERLtechtvm) [reply repositories.txt:7221]: that is my old url  i change it with my new url do i detele it . this is my new url https://github.com/ERLtechtvm/RobotControl.git

Comment (per1234) [reply repositories.txt:7221]: Ah, I see now you have been changing your username many times. So I was wrong and it is OK to remove this URL.

However, you still must fix the problem of the non-unique library name.

**Pull request closed without merging**